rotkehlchen/tests/api/test_history.py

```diff
@@ -29,6 +29,7 @@
 @pytest.mark.parametrize(
     'added_exchanges',
     [(Location.BINANCE, Location.POLONIEX, Location.BITTREX, Location.BITMEX, Location.KRAKEN)],
+    # [(Location.BINANCE, Location.BITTREX, Location.BITMEX, Location.KRAKEN)],
```

Comment:
If is not needed can be removed

Reply:
Ah fuck this was a leftover. Will remove it later